notebooks/using-mlflow/no-code-deployment/deploying_with_mlflow.ipynb

```diff
@@ -0,0 +1,469 @@
+    "    config=config,\n",
+    "    name=\"mymodel-aci-deployment\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57386544",
+   "metadata": {},
+   "source": [
+    "### Deploying models to Managed Inference\n",
+    "\n",
+    "Deployments can be generated using both the Python API for MLflow or MLflow CLI. In both cases, a JSON configuration file needs to be indicated with the details of the deployment you want to achieve. The full specification of this configuration can be found at [Managed online deployment schema (v2)](https://docs.microsoft.com/en-us/azure/machine-learning/reference-yaml-deployment-managed-online).\n",
+    "\n",
+    "#### Configuration example for an Managed Inference Service deployment (real time)\n",
+    "\n",
+    "```json\n",
+    "{\n",
+    "    \"instance_type\": \"Standard_DS2_v2\",\n",
+    "    \"instance_count\": 1,\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "Remarks:\n",
+    "- We recommend `instance_count` to be at least 3 to ensure High Availability."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dd22edac",
+   "metadata": {},
+   "source": [
+    "The following sample deploys a model to a real time Managed Inference Endpoint:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3550891c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "from mlflow.deployments import get_deploy_client\n",
+    "\n",
+    "# Create the deployment configuration.\n",
+    "deploy_config = {\n",
+    "    \"instance_type\": \"Standard_DS2_v2\",\n",
+    "    \"instance_count\": 1,\n",
+    "}\n",
+    "\n",
+    "# Write the deployment configuration into a file.\n",
+    "deployment_config_path = \"deployment_config.json\"\n",
+    "with open(deployment_config_path, \"w\") as outfile:\n",
+    "    outfile.write(json.dumps(deploy_config))\n",
+    "\n",
+    "# Set the tracking uri in the deployment client.\n",
+    "client = get_deploy_client(\"<azureml-mlflow-tracking-url>\")\n",
+    "\n",
+    "# MLflow requires the deployment configuration to be passed as a dictionary.\n",
+    "config = {\"deploy-config-file\": deployment_config_path}\n",
+    "model_name = \"mymodel\"\n",
+    "model_version = 1\n",
+    "\n",
+    "# define the model path and the name is the service name\n",
+    "# if model is not registered, it gets registered automatically and a name is autogenerated using the \"name\" parameter below\n",
+    "client.create_deployment(\n",
+    "    model_uri=f\"models:/{model_name}/{model_version}\",\n",
+    "    config=config,\n",
+    "    name=\"mymodel-mir-deployment\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fffa4635",
+   "metadata": {},
+   "source": [
+    "Remarks:\n",
+    "- Notice how `endpoint` has been specified for Managed Inference endpoints, which support multiple endpoints being deployed to the same deployment."
```

Comment:
which support multiple endpoints being deployed to the same deployment. -> which support multiple deployment being deployed to the same endpoint.

Reply:
Fixed,